### ISSUE TITLE
daemon stops unexpectedly (v0.5.0.0)

### ISSUE THREAD

**Issue by WURMiL** [wownero/wownero#186]
wownerod.exe on Win10x64 quits immediately after showing these lines without any error message even if the log level set to detailed (4):

2019-03-04 20:48:29.153 13064   INFO    global  src/daemon/main.cpp:227 Wownero 'Erotic EggplantEmoji' (v0.5.0.0-release)
2019-03-04 20:48:29.153 13064   INFO    global  src/daemon/protocol.h:53        Initializing cryptonote protocol...
2019-03-04 20:48:29.153 13064   INFO    global  src/daemon/protocol.h:58        Cryptonote protocol initialized OK

**Comment by jwinterm** [comment]
I'm aware of this. I'm getting the same behavior when building Monero from master as well, and I've tried building on two different computers now. So, I will either just wait for Monero to fix it upstream or work with them using debug build to try and figure it out.

**Comment by wowario** [comment]
You can try this...

1. in the taskbar search, type "powershell"
2. right-click on powershell and select "Run as Administrator"
3. copy and paste this `Enable-WindowsOptionalFeature -Online -FeatureName Microsoft-Windows-Subsystem-Linux` and hit enter
4. restart your computer
5. open the Microsoft Store, search for "ubuntu", select the first result, click on the Install 
6. in the taskbar search, type "ubuntu", click on ubuntu
7. type these commands:
```
  sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys B09DF0E4B0C56A94
  sudo add-apt-repository "deb http://ppa.wownero.com/ bionic main"
  sudo apt-get update
  sudo apt-get install wownero
```
8. type "wownerod" and hit enter to run daemon
9. once synced, type "wownero-wallet-cli" and enter to open wallet
10. pat yourself on the back

**Comment by WURMiL** [comment]
but why 0.4.0.0 works normally even on Win10 (x64)?

**Comment by wowario** [comment]
because there were [tons of changes](https://github.com/wownero/wownero/compare/master...release-v0.4.0.1) and now new version is broke on Windows

**Comment by wowario** [comment]
fixed with [0.5.1](https://github.com/wownero/wownero/releases/tag/v0.5.1.0)
